doc/set_version.sh

```diff
@@ -0,0 +1,7 @@
+if [ -z "${DALROOT}" ]; then
+    export DALROOT=${PREFIX}
+fi
+
+export DOC_VERSION=`python -c "import sys; import os; sys.path.append(os.path.join(os.getcwd(), '..', 'scripts')); from version import get_onedal_version; print(get_onedal_version('${DALROOT}'))"`
```

Comment:
Please address codefactor hit here and refactor your code a bit

Reply:
done